plugins/janus_recordplay.c

```diff
@@ -2618,6 +2677,98 @@ static void *janus_recordplay_playout_thread(void *data) {
 				}
 			}
 		}
+		if(data) {
+			if(data == session->dframes) {
+				/* First packet, send now */
+				/* Data recording stores recording timestamp in first 8 bytes - it follows the frame ts monotonically.
+					invariant: when = data->ts + when(initial packet) */
+				gint64 when = 0;
+				int len = data->len;
+				int offset = data->offset;
+				fseek(dfile, data->offset, SEEK_SET);
+				bytes = fread(&when, sizeof(gint64), 1, dfile);
+				when = ntohll(when); // NOTE: not currently used - playback is interested in actual data packets.
```

Comment:
Code style: comments should always be in `/*  */` format.

Reply:
Corrected.